### PR TITLE
Replace old AuthorsFormatters with the correct new ones

### LINKED ISSUE #7356: Convert layout resources to new authors formatter
While fixing the Harvard exporter, I noticed that all layout files still use the old AuthorsFormatter chaining:

https://github.com/JabRef/jabref/tree/master/src/main/resources/resource/layout

The format for author formatting has changed to https://docs.jabref.org/collaborative-work/export/customexports
 it's now seomthing like `\format[Authors(LastFirst,....),HTMLChars]`

 

JabRef version  on 

- [x] **Mandatory**: I have tested the latest development version from http://builds.jabref.org/master/ and the problem persists

All layout files must be checked and the Author and Editor Formatter changed

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fixes #7356

TODO: Add tests 

@k3KAW8Pnf7mkmdSMPHz27  Would be nice if you could take a look if I replaced them correctly

- [ ] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
I should be able to have a look at them all today

File: src/main/resources/resource/layout/openoffice/openoffice-csv.layout [hunk 1]
```diff
@@ -1 +1 @@
-\format[GetOpenOfficeType]{\entrytype},"\begin{isbn}\isbn\end{isbn}","\citationkey","\format[AuthorLastFirst,AuthorAndToSemicolonReplacer]{\author}","\format[RemoveBrackets,RemoveWhitespace]{\title}","\journal",\volume,\number,"\month","\pages",\year,"\address","\note","\url","\booktitle","\chapter","\edition","\series","\format[AuthorLastFirst,AuthorAndToSemicolonReplacer]{\editor}","\publisher","\begin{reporttype}\reporttype\end{reporttype}","\howpublished","\institution","\organization","\school","\annote","\format[Replace(\n, )]{\abstract}","\comment","\keywords","\format[FileLink(pdf)]{\file}","\key"
+\format[GetOpenOfficeType]{\entrytype},"\begin{isbn}\isbn\end{isbn}","\citationkey","\format[Authors(LastFirst,Semicolon)]{\author}","\format[RemoveBrackets,RemoveWhitespace]{\title}","\journal",\volume,\number,"\month","\pages",\year,"\address","\note","\url","\booktitle","\chapter","\edition","\series","\format[Authors(LastFirst,Semicolon]{\editor}","\publisher","\begin{reporttype}\reporttype\end{reporttype}","\howpublished","\institution","\organization","\school","\annote","\format[Replace(\n, )]{\abstract}","\comment","\keywords","\format[FileLink(pdf)]{\file}","\key"
```
Comment (k3KAW8Pnf7mkmdSMPHz27) [inline src/main/resources/resource/layout/openoffice/openoffice-csv.layout:1]: I believe there is a missing `)` in the second `Authors(...)` and I'd go with `Authors(LastFirst,FullName,Semicolon,Semicolon)` based on the previous version.

**Review by k3KAW8Pnf7mkmdSMPHz27 — COMMENTED** [review]
Given that the layout files predate the GitHub migration, I have only added comments on when I'd use something different based on the previous version. I couldn't even find a good reference for din1505, which makes my review less useful 😬 
If you agree with the suggested changes, I can push/pr based on this branch so you don't have to ctrl+f & copy-paste it for no good reason.

**Comment by Siedlerchr** [comment]
Thanks for taking a look.
Feel free to push to this branch! Your Changes seem reasonable. 
for Din 1505 I only found Wikipedia with someeyavoke

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
> Feel free to push to this branch!

I might have spoken too soon. I can only see this branch as part of JabRef/jabref.git which I don't think I have permission to, which would make sense.

I can open a PR for https://github.com/k3KAW8Pnf7mkmdSMPHz27/jabref/tree/fixlayoutauthors but at that point, it might be easier to just copy-paste it.

I'll play around with my settings and see if anything changes, but I don't think I have made a mistake.

**Comment by Siedlerchr** [comment]
@k3KAW8Pnf7mkmdSMPHz27  I added you as contributor to this project (thought you were already) so you should be able to push to that branch

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
Also, there are some issues with `Authors.java`. It should produce the wrong output for two authors using `oxford` and it uses the form `von Last Jr., First` rather than `von Last, Jr., First`.
I planned to look at it in the follow-up PR of https://github.com/JabRef/jabref/pull/7301#discussion_r559259967, but perhaps it needs attention sooner.

**Review by koppor — APPROVED** [review]
(DevCall) We accept that we did not add tests here.

**Pull request merged**